recipes/easylzma/all/conanfile.py

```diff
@@ -57,6 +62,9 @@ def package(self):
         self.copy("easylzma/*", dst="include", src=os.path.join(self._source_subfolder, "src"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.libs = [self._libname]
```

Comment:
Why do you prefer doing it explicitly like thuis instead of using the collect_libs() tool?

Reply:
It does not matter in this case since their is only one library.
But in the general case with more then one libraries, their might be dependencies between them.